b1ddi/data_source_ipam_next_available_ip.go

```diff
@@ -0,0 +1,253 @@
+package b1ddi
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	b1ddiclient "github.com/infobloxopen/b1ddi-go-client/client"
+	"github.com/infobloxopen/b1ddi-go-client/ipamsvc/address"
+	"strings"
+)
+
+func dataSourceIpamsvcNaIP() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceIpamsvcNaIPRead,
+		Schema: map[string]*schema.Schema{
+
+			// The address in form "a.b.c.d".
+			// Required: true
+			"address": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "The address in form \"a.b.c.d\".",
+			},
+
+			// The description for the address object. May contain 0 to 1024 characters. Can include UTF-8.
+			"comment": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "The description for the address object. May contain 0 to 1024 characters. Can include UTF-8.",
+			},
+
+			// Time when the object has been created.
+			// Read Only: true
+			// Format: date-time
+			"created_at": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Time when the object has been created.",
+			},
+
+			// The DHCP information associated with this object.
+			// Read Only: true
+			"dhcp_info": {
+				Type:        schema.TypeList,
+				Elem:        schemaIpamsvcDHCPInfo(),
+				Computed:    true,
+				Description: "The DHCP information associated with this object.",
+			},
+
+			// The resource identifier.
+			"host": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "The resource identifier.",
```

Comment:
For all those fields containing identifiers, it'll be better to provide an accurate description and not `"The resource identifier."`.

Reply:
I have updated the datasource implementation. We no longer need this field in the schema.